botx/models/system_events/cts_logout.py

```diff
@@ -0,0 +1,57 @@
+from dataclasses import dataclass
+from typing import Any, Dict
+from uuid import UUID
+
+from pydantic import Field
+
+from botx.models.api_base import VerifiedPayloadBaseModel
+from botx.models.base_command import (
+    BaseBotAPIContext,
+    BotAPIBaseCommand,
+    BotCommandBase,
+)
+from botx.models.enums import BotAPICommandTypes
+
+try:
+    from typing import Literal
+except ImportError:
+    from typing_extensions import Literal  # type: ignore  # noqa: WPS440
+
+
+@dataclass
+class CTSLogoutEvent(BotCommandBase):
+    """Event `system:cts_logout`.
+
+    Attributes:
+        user_huid: user ID.
+        cts_id: ID of the server the user is logged out from.
```

Comment:
We can't use this field, maybe just drop it?

Reply:
done